automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoApiClient.scala

```diff
@@ -136,10 +133,7 @@ object LeonardoApiClient {
           headers = Headers.of(authHeader),
           uri = rootUri.withPath(s"/api/google/v1/runtimes/${googleProject.value}/${runtimeName.asString}/start")
         )
-      )(resp =>
-        resp.bodyText.compile.string
-          .flatMap(body => IO.raiseError(RestError(resp.status, body)))
-      )
+      )(onError(s"Failed to start runtime ${googleProject.value}/${runtimeName.asString}"))
```

Comment:
I liked raising the error here, otherwise its hard to tell why the call actually failed (namely when I was debugging those SAM errors, the error message was hidden with this code)